src/stories/components/JsPsych/JsPsych.js

```diff
@@ -0,0 +1,240 @@
+    jsPsychLibrary.async = true;
+    jsPsychStyleSheet.async = true;
+    htmlKeyboardResponsePlugin.async = true;
+    imageKeyboardResponsePlugin.async = true;
+    document.body.appendChild(jsPsychLibrary);
+    document.body.appendChild(jsPsychStyleSheet);
+    document.body.appendChild(htmlKeyboardResponsePlugin);
+    document.body.appendChild(imageKeyboardResponsePlugin);
+
+    // Load participantId, experimentId, and 400 stimuli urls
+    observeStimuliCompletion(participantId, experimentId, setStimuliUrls, errorLoadingJsPsych);
+
+    return () => {
+      document.body.removeChild(jsPsychLibrary);
+      document.body.removeChild(jsPsychStyleSheet);
+      document.body.removeChild(htmlKeyboardResponsePlugin);
+      document.body.removeChild(imageKeyboardResponsePlugin);
+    };
+  }, []);
+
+  useEffect(() => {
+    if (stimuliUrls.length > 0) {
+      setReady(true);
+    }
+  }, [stimuliUrls]);
+
+  // Error handler that prompts the participant to re-click experiment.
+  const errorLoadingJsPsych = (errorCode) => {
+    window.alert('Something went wrong. Please click on experiment again.' +
+      ' Error code: ' + errorCode);
+  };
+
+  // Have to wait for the plugin scripts to be loaded asynchronously before
+  // running the experiment; otherwise it crashes.
+  setTimeout(() => {
+    // Implementation of previous team timeline & trial logic -------------------------- START
+    /* create timeline */
+    const timeline = [];
+
+    /* number of trials */
+    // TODO: Adjust NUMBER_OF_TRIALS back to 199
+    // NOTE: Adjust line below to shorten the number of trials. 199 will go through all 200 iterations.
+    const NUMBER_OF_TRIALS = 5;
+
+    /* define instructions */
+    const instructions = {
+      type: 'html-keyboard-response',
+      stimulus: function() {
+          return "<h1>Instruction</h1>" +
+            "<p>In this experiment, two images will be shown " +
+            "on the screen. Choose the image that looks more like you. </p>" +
+            "<p>Press the letter <strong>E</strong> on the keyboard to select the image on the left.</p>" +
+            "<p>Press the letter <strong>I</strong> on the keyboard to select the image on the right.</p> " +
+            "<p></p>" +
+            "<div style='width: 900px; margin: auto;'>" +
+            // TODO: Figure out how to properly render example images. Both implementations below do not render local images.
+            //        We are able to render via HTTPs URLs but having issues with loading images locally.
+            "<div style='float: left;'><img width='300' src={require('../../../images/example1.png')} alt='Error loading example 1'></img>" +
+            "<p class='small'><strong>Press the E key</strong></p></div>" +
+            "<div class='float: right;'><img width='300' src={ example2 } alt='Error loading example 2'></img>" +
```

Comment:
This is exactly where I've been scratching my head all day. Line 91 - 93. I have been unsuccessful at finding a way to solve this issue. Rendering an image locally. Line 91 and Line 93 are using two different methods of rendering images, both of which do not work.

Reply:
If the import syntax I suggested above doesn't work, then I would say let's just upload the example images to out cloud storage and pull them from there using HTTPs URLs.

Reply:
Sounds good to me! I believe that solution will work.

Reply:
Just uploaded images to our cloud storage and images render properly now.